apps/mkt/components/automation-carousel/automation-icon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { TablerIconsProps } from "@tabler/icons-react";
+
+interface AutomationIconProps extends TablerIconsProps {
+  IconComponent: React.FC<TablerIconsProps>;
+}
+
+const AutomationIcon: React.FC<AutomationIconProps> = ({
+  IconComponent,
+  color = "#fff",
+  stroke = 1.5,
+  ...props
+}) => {
```

Comment:
This way works, and it's fine, but just for standardization, we usually just declare the type like this:

```suggestion
const AutomationIcon = ({
  IconComponent,
  color = "#fff",
  stroke = 1.5,
  ...props
}: AutomationIconProps) => {
```